source/index.pug

```diff
@@ -0,0 +1,19 @@
+doctype html
+
+html(lang="en")
+  head
+    meta(charset="UTF-8")
+    meta(
+      name="viewport",
+      content="width=device-width, initial-scale=1.0, user-scalable=no, shrink-to-fit=no"
+    )
+
+    link(href="index.scss", rel="stylesheet")
+
+    title Newsweek - News, Analysis, Politics, Business, Technology
+
+    script(src="index.js")
+
+  body
+    include _layout
+
```

Comment:
- [ ] Checking with the WC3 validator (copying the code that you have from the generated HTML file) you have some errors. Please, fix them.
> ![image](https://user-images.githubusercontent.com/57414789/81484085-7b0c0380-9208-11ea-8f09-49083bd8de58.png)

- [ ] The navbar should include the weather info.

- [ ] In the original webpage they use links in most of the text and images. Please fix that.

- [ ] Phone version. Below the top stories should be featured stories, then opinions, then more stories, and so on. Please check that.

- [ ] More stories section has a different layout for the phone version.
> Your website
![image](https://user-images.githubusercontent.com/57414789/81484307-28334b80-920a-11ea-85c4-832c9db26e4c.png)

> Original
![image](https://user-images.githubusercontent.com/57414789/81484314-33867700-920a-11ea-9802-65addd482633.png)

- [ ] "In the magazine" section is different from the original.
> Screenshot from original webpage.
![image](https://user-images.githubusercontent.com/57414789/81484151-20bf7280-9209-11ea-83eb-9c8eeed7d99c.png)

- [ ] Editor's pick's desktop version should have 4 columns.
> Screenshot from the original webpage
![image](https://user-images.githubusercontent.com/57414789/81484189-58c6b580-9209-11ea-9b91-711574de2908.png)

- The subscribe section should display the images together, not separately. Kindly check the layout because the letters are not fully displayed.
> Screenshot
![image](https://user-images.githubusercontent.com/57414789/81484406-c9ba9d00-920a-11ea-85dc-951f530ce50f.png)

- [ ] The footer should be responsive.